### PR TITLE
Add D8 and WordPress info for wkhtmltopdf usage [wip]

### LINKED ISSUE #5330: External Libraries WKHTMLtoPDF Drupal version
Re: [External Libraries on Pantheon](https://pantheon.io/docs/external-libraries)

Priority: Low

## Issue Description:
WKHTMLtoPDF Drupal installation instructions are for Drupal 7 sites, using the Print module, only. Drupal 8 sites can use the [WKTHMLtoPDF module](https://www.drupal.org/project/wkhtmltopdf).

## Suggested Resolution
Modify instructions to include tabs for Drupal 7 and Drupal 8.

### PR DESCRIPTION
**Pull request by sarahg**
Closes #5330 

## Effect
PR includes the following changes:
- Add D8 and WordPress info for wkhtmltopdf usage

## Remaining Work
- [ ] Test both D8 and WP options a bit more and add details on configuration for both
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
Unfortunately the proposed WP plugin was a bust. There's still no working plugin for WordPress to utilize wkhtmltopdf locally.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5342 | Difference |
| --- | --- | --- | --- |
| Performance Score | `80` | `62` | `-18` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `96` | `-2` |

Lighthouse Audit Reports:

- [`pr-5342` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18144/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18144/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5342 | Difference |
| --- | --- | --- | --- |
| Performance Score | `48` | `47` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5342` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18526/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18526/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5342 | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `41` | `-10` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5342` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19094/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19094/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5342 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `54` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5342` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19102/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19102/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/external-libraries.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ If your WordPress site uses Composer, consider the [PHP WkHtmlToPdf](https://git
 
 </TabList>
 
+### Unable to Generate PDF File
```
Comment (EdwardAngert) [inline source/content/external-libraries.md]: @spheromak @winmillwill thanks for your help hunting this down! Can I please get your 👀 and 👍 on this section?

**Pull request merged**